tests/easyInterface/Diffraction/test_Interface.py

```diff
@@ -93,6 +94,44 @@ def test_setInfoDict(cal):
     assert 'Fe3O4' == cal.project_dict['info']['name']
 
 
+def test_setPhase(cal):
+    phase = cal.getPhase('Fe3O4')
+    cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value = 2
+    cal.setPhase(phase)
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value == 8.36212
+
+
+def test_setPhaseRefine(cal):
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].refine is True
+    cal.setPhaseRefine('Fe3O4', ['cell', 'length_a'], False)
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].refine is False
+
+
+def test_setPhaseValue(cal):
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value == 8.36212
+    cal.setPhaseValue('Fe3O4', ['cell', 'length_a'], 5)
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value == 5
+
+
+def test_setPhase(cal):
+    phase = cal.getPhase('Fe3O4')
+    cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value = 2
+    cal.setPhase(phase)
+    assert cal.project_dict['phases']['Fe3O4']['cell']['length_a'].value == 8.36212
```

Comment:
This function has already been defined, and with the same content, on line 97-101.

Reply:
Removed. Total face palm